### PR TITLE
fix: delay evaluation to finish preprocessing

### PR DESCRIPTION
**Pull request by bajtos**
A temporary hacky fix for #64. See also #209.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -85,6 +85,14 @@ export const startEvaluate = async ({
       return
     }
 
+    // TODO: Fix this properly and implement a signalling mechanism allowing the "preprocess" step
+    // to notify the "evaluate" when the preprocessing is done, so that we don't have to use a timer
+    // here. See also https://github.com/filecoin-station/spark-evaluate/issues/64
+    const delay = PREPROCESS_DELAY + 60_000
```
Comment (juliangruber) [inline index.js:91]: can we define this in the file header?

```js
const EVALUATE_DELAY = PREPROCESS_DELAY + 60_000
```

**Review by juliangruber — CHANGES_REQUESTED** [review]
Good idea!

File: index.js (hunk 1 above)
Comment (bajtos) [reply index.js]: 👍🏻 

See [c014df3](https://github.com/filecoin-station/spark-evaluate/pull/224/commits/c014df3c90d631743146944294adbad57c5476c4)

**Review by juliangruber — APPROVED** [review]

**Comment by bajtos** [comment]
Fix: https://github.com/filecoin-station/spark-evaluate/pull/230

**Pull request merged**